templates/deploy-ansible-service-broker.template.yaml

```diff
@@ -199,17 +200,25 @@ objects:
             enabled: ${ENABLE_BASIC_AUTH}
 
 - apiVersion: servicecatalog.k8s.io/v1alpha1
-  kind: Broker
+  kind: ${BROKER_KIND}
   metadata:
     name: ansible-service-broker
   spec:
     url: ${ASB_SCHEME}://asb-1338-ansible-service-broker.${ROUTING_SUFFIX}
     authInfo:
-      basicAuthSecret:
-        namespace: ansible-service-broker
-        name: asb-auth-secret
+      ${{BROKER_AUTH}}
 
 parameters:
+- description: Service Broker kind. Newer service-catalogs use ServiceBroker
+  displayname: Service Broker kind. Newer service-catalogs use ServiceBroker
+  name: BROKER_KIND
+  value: Broker
```

Comment:
3.7 requires ServiceBroker.  Should we make that the default since we need to base off of 3.7 anyway?

Reply:
3.7 currently does not use v0.0.17 so still expects 'Broker'.

When it changes we should probably change the defaults to be what we need for current.